packages/p2p/src/components/online-status/online-status.scss

```diff
@@ -0,0 +1,42 @@
+.online-status {
+    &__icon {
+        width: 1em;
+        height: 1em;
```

Comment:
@adrienne-deriv, Is it possible to use `rem` instead `em`

Reply:
gotchaa